docs/config.md

```diff
@@ -227,6 +227,27 @@ cpu_only_composing_models: <comma-delimited-string-list>
 # Enables the searching of request rate (instead of concurrency)
 [ request_rate_search_enable: <bool> | default: false]
 
+# Minimum percentage of the search space to profile when using Optuna
+[ min_percentage_of_search_space: <int> | default: 5]
+
+# Maximum percentage of the search space to profile when using Optuna
+[ max_percentage_of_search_space: <int> | default: 10]
```

Comment:
[tgerdesnv](https://github.com/tgerdesnv) [31 minutes ago](https://github.com/triton-inference-server/model_analyzer/pull/894#discussion_r1629496624)
5-10% seems like a very small band. Do your experiments show that <5% is generally bad and >10% is overkill?

Member
Author
@[nv-braf](https://github.com/nv-braf) nv-braf [17 minutes ago](https://github.com/triton-inference-server/model_analyzer/pull/894#discussion_r1629517961) • 
These are fairly arbitrary, but they seemed like a good starting point based on my initial testing.
I'm not sure how well this scales though once we go BLS/Ensemble though as the space will grow exponentially and even 5% would result in hundreds of configs.